rpe/resources/gcp.py

```diff
@@ -917,3 +917,22 @@ def _get_request_args(self):
             'location': self._resource_data['location'],
             'view': 'JOB_VIEW_DESCRIPTION'
         }
+
+class GcpMemorystoreRedis(GoogleAPIResource):
```

Comment:
This is an odd one, they lump 2 apis under the name _Memorystore_. The existing naming convention is Gcp[APIName][ResourceType]. I think this should change to GcpRedisInstance

Reply:
Good to know 😄. Updated